bulbs/content/tasks.py

```diff
@@ -199,7 +201,9 @@ def post_to_instant_articles_api(content_pk):
                 source,
                 fb_page_id,
                 fb_api_url,
-                fb_token_path)
+                fb_token_path,
+                fb_dev_mode,
```

Comment:
Yeah, that's true. Wouldn't that just always post then, assuming we're setting `fb_dev_mode` to True in base settings? Maybe we should nix the environment flag and just add a boolean `FACEBOOK_POST_TO_IA` flag, so we don't have to rely on these variables that we're posting to determine if we should send it or not?